operations/system_management_health/Grafana_Dashboards_by_Component.md

```diff
@@ -37,6 +37,13 @@ The panels interact with configured data sources, including the following:
 - Main
 - Clusters
 
+## Goss tests
+
+- Product test results
+- Suite test results
+- Test results
+- Node-wise test results
```

Comment:
This may or may not need to be changed based on my earlier comment where I asked what ```node-wise``` means.